diff --git a/target/linux/rockchip/armv8/config-6.12 b/target/linux/rockchip/armv8/config-6.12
index 6fb6dae..7c6a418 100644
--- a/target/linux/rockchip/armv8/config-6.12
+++ b/target/linux/rockchip/armv8/config-6.12
@@ -725,6 +725,7 @@ CONFIG_USB_PHY=y
 CONFIG_USB_ROLE_SWITCH=y
 CONFIG_USB_STORAGE=y
 CONFIG_USB_SUPPORT=y
+CONFIG_USB_UAS=y
 CONFIG_USB_ULPI=y
 CONFIG_USB_ULPI_BUS=y
 CONFIG_USB_ULPI_VIEWPORT=y
