--- a/utils/qemu/Makefile
+++ b/utils/qemu/Makefile
@@ -31,7 +31,7 @@
 include ../../lang/python/python3-host.mk
 
 QEMU_DEPS_IN_GUEST := @(TARGET_x86||TARGET_x86_64||TARGET_armsr||TARGET_malta)
-QEMU_DEPS_IN_HOST := @(TARGET_x86_64||TARGET_armsr_armv8||TARGET_sunxi)
+QEMU_DEPS_IN_HOST := @(TARGET_x86_64||TARGET_armsr_armv8||TARGET_rockchip_armv8||TARGET_sunxi)
 QEMU_DEPS_IN_HOST += +libstdcpp
 QEMU_DEPS_IN_HOST += $(ICONV_DEPENDS)
 
@@ -211,6 +211,8 @@
   define Package/qemu-$(1)/install
 	$(INSTALL_DIR) $$(1)/usr/bin
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/$(call qemu-prog,$(1)) $$(1)/usr/bin
+	$(if $(filter aarch64-softmmu,$(1)),$(INSTALL_DIR) $$(1)/usr/share/qemu)
+	$(if $(filter aarch64-softmmu,$(1)),-$(CP) $(PKG_INSTALL_DIR)/usr/share/qemu/edk2-aarch64-code.fd $(PKG_INSTALL_DIR)/usr/share/qemu/edk2-arm-vars.fd $$(1)/usr/share/qemu/)
 	$(foreach f,$(qemu-target-$(1)-extra-files),
 		$(INSTALL_DIR) $$(1)/$(dir $(f))
 		$(CP) $(PKG_INSTALL_DIR)/$(f) $$(1)/$(f)
